pennylane/hf/integrals.py

```diff
@@ -308,6 +308,207 @@ def overlap_integral(*args):
     return overlap_integral
 
 
+def _hermite_moment(alpha, beta, t, e, r):
```

Comment:
The new integrals are from a previous, still open, [PR](https://github.com/PennyLaneAI/pennylane/pull/2166). Please skip reviewing this file!